params/config.go

```
@@ -1063,9 +1066,18 @@ type AstriaErc20AssetConfig struct {
 	ContractPrecision uint16         `json:"contractPrecision"`
 }
 
-func (abc *AstriaBridgeAddressConfig) Validate() error {
-	if len(abc.BridgeAddress) != 20 {
-		return fmt.Errorf("bridge address must be 20 bytes")
+func (abc *AstriaBridgeAddressConfig) Validate(hrpPrefix string) error {
+	prefix, byteAddress, err := bech32.Decode(abc.BridgeAddress)
+	byteAddress, _ = bech32.ConvertBits(byteAddress, 5, 8, false)
```

don't ignore the error here?